how-tos/setup-staging-release.md

```diff
@@ -39,67 +39,14 @@ hg -R build/jamun push
 - dev [Ship-it](https://ship-it-dev.allizom.org/)
 - staging [release runner](https://dxr.mozilla.org/build-central/rev/5f83e0516fc449586bbce4db4eb759f6cede8781/puppet/manifests/moco-nodes.pp#633)
 - release automation notifications [group](https://groups.google.com/a/mozilla.com/forum/?hl=en#!forum/release-automation-notifications-dev) and `#release-notifications-dev` IRC channel
-- ~~staging [balrog](https://balrog-admin.stage.mozaws.net/)~~ Spawn your own balrog instance. See dedicated appendix below.
-- point that instance in all configs [mozharnss configs](https://dxr.mozilla.org/mozilla-central/source/testing/mozharness/configs/releases/) that need it. Sample [here](https://dxr.mozilla.org/mozilla-central/rev/7d2e89fb92331d7e4296391213c1e63db628e046/testing/mozharness/configs/releases/dev_updates_firefox_beta.py#23)
 
 ## Staging configs
 
 - release runner will consume [project branch configs](https://dxr.mozilla.org/build-central/rev/92614acc90330edf360d97d8575b7e917ddc43b2/buildbot-configs/mozilla/project_branches.py#114)
 - mozharness [configs](https://dxr.mozilla.org/mozilla-central/source/testing/mozharness/configs/releases/) - all configs with `dev` prefix
-- in order to avoid pushing patcher configs changes/tags to *real* [tools](http://hg.mozilla.org/build/tools/) repo, please point to a user fork. You can reuse rail's [tools repo](https://hg.mozilla.org/users/raliiev_mozilla.com/tools-fake/)
-
-A note on creating custom `tools` repo.
-Rather than reusing an existing fellow RelEnger `tools` repo, you should clone your own to avoid missing permissions sort-of-issues.
-That can be done in the following way (more on this [here](https://wiki.mozilla.org/Release:Release_Automation_on_Mercurial:Staging_Specific_Notes#Setup_staging_repos)):
-
-```sh
-ssh hg.mozilla.org clone tools-fake build/tools
-# Will show: Please wait.  Cloning /build/tools to /users/jlorenzo_mozilla.com/tools-fake
-```
+- in order to avoid pushing patcher configs changes/tags to *real* [tools](http://hg.mozilla.org/build/tools/) repo, we have changed the buildbot and in tree configs to point to the staging tools repo https://hg.mozilla.org/users/stage-ffxbld/tools) 
```

Comment:
Nit: maybe add the https://hg.mozilla.org/users/stage-ffxbld/tool as a href to the "tools" for consistency?